.github/workflows/aks-deploy.yaml

```diff
@@ -301,48 +307,52 @@ jobs:
           docker build ${{ inputs.dockerFilePath }} ${{ steps.getsecrets.outputs.buildArguments }} -t "${{ env.containerRegistry  }}/${{ inputs.dockerImageName }}:${{ inputs.dockerImageTag }}"
           docker push "${{ env.containerRegistry }}/${{ inputs.dockerImageName }}:${{ inputs.dockerImageTag }}"
 
-      - name: Create valid release-name
-        id: generate-release-name
-        run: |
-          release=${{ env.appName }}-${{ github.sha }}
-          release=${release::53}
-          release=$(echo ${release//[!0-9a-zA-Z]/-} | tr '[:upper:]' '[:lower:]' | sed -e 's/^-/z/' -e 's/-$/z/')
-          echo "release=$release" >> $GITHUB_ENV
-
       - name: Create values override file
         run: |
           WEB_AUTHENTICATION="${{ inputs.webAuthentication }}"
-          ADMIN_INGRESS_WHITELIST="${{ inputs.adminIngressWhitelist }}"
-          INGRESS_WHITELIST="${{ inputs.ingressWhitelist }}"
-          if [[ ${WEB_AUTHENTICATION} == "true" ]] ; then
+          INGRESS_WHITELIST_ENV="${{ env.ingressWhitelist }}"
+          ADMIN_INGRESS_WHITELIST_ENV="${{ env.adminIngressWhitelist }}"
+          if [[ -n "${INGRESS_WHITELIST_ENV}" ]] ; then
+            INGRESS_WHITELIST="${{ env.ingressWhitelist }}"
+          else
+            INGRESS_WHITELIST="${{ inputs.ingressWhitelist }}"
+          fi
+
+          if [[ -n "${ADMIN_INGRESS_WHITELIST_ENV}" ]] ; then
+            ADMIN_INGRESS_WHITELIST="${{ env.adminIngressWhitelist }}"
+          else
+            ADMIN_INGRESS_WHITELIST="${{ inputs.adminIngressWhitelist }}"
+          fi
+
+          if [[ "${WEB_AUTHENTICATION}" == "true" ]] ; then
           cat << EOF >> values-override.yaml
           ingress:
             annotations:
               nginx.ingress.kubernetes.io/limit-whitelist: "207.67.20.252"
-              nginx.ingress.kubernetes.io/whitelist-source-range: "${{ inputs.ingressWhitelist }}"
+              nginx.ingress.kubernetes.io/whitelist-source-range: "${INGRESS_WHITELIST}"
               nginx.ingress.kubernetes.io/auth-type: "basic"
               nginx.ingress.kubernetes.io/auth-secret: "${{ env.appName }}-basic-auth"
               nginx.ingress.kubernetes.io/auth-realm: "Authentication Required"
           adminingress:
             annotations:
-              nginx.ingress.kubernetes.io/whitelist-source-range: "${{ inputs.adminIngressWhitelist }}"
+              nginx.ingress.kubernetes.io/whitelist-source-range: "${ADMIN_INGRESS_WHITELIST}"
           EOF
-          elif [[ ${ADMIN_INGRESS_WHITELIST} != "207.67.20.252" ]] || [[ ${INGRESS_WHITELIST} != "0.0.0.0/0" ]]; then
+          elif [[ "${ADMIN_INGRESS_WHITELIST}" != "207.67.20.252" ]] || [[ "${INGRESS_WHITELIST}" != "0.0.0.0/0" ]]; then
```

Comment:
Should we put this in an org level secret?

Reply:
Putting them in a org level secret might cause more confusion than what it is worth. The default values in the inputs already account for an org-level standard. 

Though, checking whether the default value is being used or not in that if/else condition could be improved most likely.